src/observer.js

```diff
@@ -29,21 +38,24 @@ Observer.prototype = {
 
   destroy() {
     const container = this.container
-    const viewport = viewports.get(container)
+    const viewport = this.observerCollection.viewports.get(container)
 
     if (viewport) {
       const viewportObservers = viewport.observers
       const observerIndex = viewportObservers.indexOf(this)
       if (observerIndex > -1) viewportObservers.splice(observerIndex, 1)
       if (!viewportObservers.length) {
         viewport.destroy()
-        viewports.delete(container)
+        this.observerCollection.viewports.delete(container)
       }
     }
   }
 }
 
+export { ObserverCollection }
+
 // Internally track all viewports so we only have 1 set of event listeners per container
-const viewports = new Map()
+const observerCollection = new ObserverCollection({ handleScrollResize: h => h })
```

Comment:
Maybe name it `defaultObserverCollection`.  I think handleScrollResize shouldn't be defined at all here.